app/models/contributor.rb

```diff
@@ -146,7 +146,7 @@ def tags?
   end
 
   def recent_replies
-    result = replies.includes(:request, :sender).reorder(created_at: :desc)
+    result = replies.includes(:request).reorder(created_at: :desc)
```

Comment:
isn't that needed for this line?
https://github.com/tactilenews/100eyes/blob/main/app/components/requests_feed/requests_feed.html.erb#L22